### ISSUE TITLE
anaconda-navigator unreadable with dark theme

### ISSUE THREAD

**Issue by ndc33** [ContinuumIO/anaconda-issues#744]
running kde plasma 5 breeze dark on opensuse leap. Text is presented in white with beige background makes it impossible to read/use. (ps i also fail to get icon/menu entry as reported in other bug reports)

**Comment by goanpeca** [comment]
could you please paste a screenshot?

**Comment by stephenakearns** [comment]
Stale issue - @ndc33 - if you have time to post screenshot, please reopen. thx.

**Comment by jat255** [comment]
This is definitely still an issue, so I'm not sure if the original poster ever worked around this, or what. I'm using breeze-dark theme on KDE (Arch linux). The main navigator screen is readable, but the Environments screen is very much not so:

![image](https://user-images.githubusercontent.com/1278301/27600577-6b99d470-5b3a-11e7-9b94-bed5acdbf64d.png)

**Comment by sossaman** [comment]
There is the same problem on Kubuntu and KDE Neon with a dark theme.

**Comment by goanpeca** [comment]
@sossaman thanks for pointing this out, fixing to have the right look even on dark theme. However, there are no plans to create a Dark Theme per se.

**Comment by sandershihacker** [comment]
Is this fixed?

**Comment by goanpeca** [comment]
Working on this for next release!

**Comment by goanpeca** [comment]
This has been fixed since version 1.8.4. 

---

Please remember to update to the latest version of Navigator to include
the latest fixes.

Open a terminal (on Linux or Mac) or the Anaconda Command Prompt (on windows)
and type:

```
conda update conda
conda update anaconda-navigator
conda update navigator-updater
```
